components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -239,6 +239,21 @@ Projects can be managed via the Projects list under the **Settings** tab and con
 [assigning ingested resources to projects]({{< relref "iam-v2-guide#assigning-ingested-resources-to-projects" >}}),
 so they are not relevant when assigning IAM resources such as teams or roles.
 
+#### Configuring Project Limit
+
+By default, Chef Automate is currently limited to six projects. If you would like to increase that limit, you will need to do so using the Chef Automate CLI:
+
+```
+cat << EOF > authz.toml
+[auth_z.v1.sys.service]
+project_limit = <desired-max-projects>
+EOF
+```
+
+`chef-automate config patch authz.toml`
```

Comment:
Is this line supposed to be the second step in committing the changes? An additional line explaining that may be needed between the file modification above and this line.
For example:

> Then, run `chef-automate configure patch authz.toml` to update the existing Chef Automate configuration.